app/src/frontend/pages/welcome.css

```diff
@@ -6,8 +6,10 @@
     z-index: 10000;
     top: 0;
     width: 100%;
+    max-height: 100%;
     border-radius: 0;
     padding: 1.5em 2.5em 2.5em;
+    overflow-y: scroll;
```

Comment:
I sometimes confuse `overflow: auto` with `scroll` - does `scroll` force a (greyed-out) scrollbar in some browsers, even when there's no overflow? I think `auto` is the one that will enable scrolling if necessary and have no scrollbar otherwise.

Reply:
I think auto will be more appropriate, I was even thinking about it today after I saw that in Chrome